app/controllers/api/subcollections/container_images.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
```

Comment:
Is this still needed?

Reply:
This file is not relevant now that I changed this to be a ContainerImage resource_action and not a subcollection_action